xmlsec.yaml

```
@@ -0,0 +1,99 @@
+package:
+  name: xmlsec
+  version: 1.3.1
+  epoch: 0
+  description: C based implementation for XML Signature Syntax and Processing and XML Encryption Syntax and Processing
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - libxml2
+      - libxml2-dev
+      - libxml2-py3
+      - libxml2-utils
+      - linux-headers
+      - wget
+      - make
+      - pkgconf
+      - pkgconf-dev
+      - binutils
+      - scanelf
+      - libtool
+      - libltdl
+      - libxslt-dev
+      - openssl-dev
+      - libssl3
+      - nss-db
+      - libgcrypt-dev
+      - python3
+      - python3-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: '_'
+    to: mangled-package-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/lsh123/xmlsec
+      expected-commit: 3b67fdb38958c8386269d85cc0bbef9a39498ff0
+      tag: xmlsec_${{vars.mangled-package-version}}
+
+  - name: Autogen
+    runs: |
+      git clean -fdx . && NOCONFIGURE=1
+      ./autogen.sh \
+        --prefix=/usr \
+        --with-openssl=/usr \
+        --enable-static \
+        --enable-static-linking \
+        --without-gnutls \
+        --without-gcrypt \
+        --without-nss \
+        --with-default-crypto=openssl \
+      && make \
+      && make -k check
+
+  - name: Install
+    runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin/
+      make -j$(nproc) DESTDIR="${{targets.destdir}}" install
+
+  - uses: strip
+
+subpackages:
+  - name: xmlsec-openssl
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib
+          mv ${{targets.destdir}}/usr/lib/libxmlsec1-openssl.so* ${{targets.subpkgdir}}/usr/lib
+    description: xmlsec openssl plugin
+
+  - name: xmlsec-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+    description: xmlsec dev
+
+  - name: xmlsec-doc
+    pipeline:
+      - uses: split/manpages
+    description: xmlsec manpages
+
+update:
+  enabled: true
```

```suggestion
  enabled: true
  version-separator: _
```
Adding this should work.  I just did a quick check using
```
wolfictl check update  --override-version 0.0.1 xmlsec.yaml
```

---

I did notice that there's a new `version-transform` config available [example](https://github.com/wolfi-dev/os/blob/0ca90c76b501d45c35e13e74049a54c18c39bfb3/openjdk-20.yaml#L221C1-L223C19),  in the update config but I don't think it's wired up correctly yet. I can help do that early next week unless it's already being looked at.  Once that does work we can switch all places that use `version-separator` config to `version-transform`.  

For now the suggestion above should make CI green and mean auto updates work.

---

`version-transform` should be working fine.  What issues are you seeing with it?